### ISSUE TITLE
Terraform Cloud - Unable to modify projects deployed by foundation

### ISSUE THREAD

**Issue by clement-chaneching** [terraform-google-modules/terraform-example-foundation#1121]
### TL;DR

I have deployed the foudation using terraform cloud.
It works fine and I can deploy the projects.
However, for the migration of my existing projects, I have other terraform cloud workspaces that needs to deploy GCP resources inside those projects.

So i m getting 2 errors: 
- I cannot manage my old workspaces to use the workload identity federation (even though I have set the variables (TFC_GCP_PROVIDER_AUTH, TFC_GCP_WORKLOAD_PROVIDER_NAME,..) 
Its throwing :
`Error: Attempted to load application default credentials since neither `credentials` nor `access_token` was set in the provider block. No credentials loaded. To use your gcloud credentials, run 'gcloud auth application-default login'`
I dont have any assertion restriction, so I guess it should allow every worskspace within my terraform cloud organization.

- When using my global owner account with the GOOGLE_CREDENTIALS, it throws me : 
`Error: Request `Enable Project Service "run.googleapis.com" for project "prj-fdn-d-test-x7l5"` returned error: Batch request and retried single request "Enable Project Service \"run.googleapis.com\" for project \"prj-fdn-d-test-x7l5\"" both failed. Final error: failed to enable services: failed on request preconditions: googleapi: Error 403: Permission denied to enable service [run.googleapis.com] Help Token: AX4KC-hIF73LL5CLIGrp_8cux8WJBTwjFIAWA2xsqO79T9LzcV8i0PGv4XW30q5Lx2m6mei3n48DCcMD9EAF1A0HDRfOJ3FiJEhf7F6Ul2HnSzmg Details: [ { "@type": "type.googleapis.com/google.rpc.PreconditionFailure", "violations": [ { "subject": "?error_code=110002\u0026service=serviceusage.googleapis.com\u0026permission=serviceusage.services.enable\u0026resource=prj-fdn-d-oth-test-prj-x7l5", "type": "googleapis.com" } ] }, { "@type": "type.googleapis.com/google.rpc.ErrorInfo", "domain": "serviceusage.googleapis.com", "metadata": { "permission": "serviceusage.services.enable", "resource": "prj-fdn-d-test-x7l5", "service": "serviceusage.googleapis.com" }, "reason": "AUTH_PERMISSION_DENIED" } ] , forbidden
`

Are there restrictions on projects deployed in the foundation using Terraform cloud that I have missed? I have removed all the IAM org policies for now, and I m using a floating project for testing...

### Expected behavior

My global admin account should still be able to deploy resources in projects deployed by foundation.
My other TF Cloud workspaces should be able to use the workload identity federation.

### Terraform Configuration

```hcl
Used the default Tf cloud files provided.
```

### Terraform Version

```sh
1.7.3
```
